source/install/requirements/requirements.rst

```diff
@@ -1,30 +1,74 @@
 
-You can check your `CUDA Properties here <https://github.com/tpruvot/ccminer/wiki/Compatibility>`_ or on the `NVIDIA dev page <https://developer.nvidia.com/cuda-gpus>`_.
+  In case you do not have a CUDA GPU, you can use the draft meshing option which uses the CPU for meshing.
 
-Install the `latest NVIDIA drivers <https://www.nvidia.com/Download/index.aspx>`_.
+Here are the minimum requirements for Meshroom:
 
-**Note**
-: If you do not have a CUDA GPU, you can use the
-draft meshing option
-which uses the CPU for meshing.
++--------------------------------------------------------------------------+
+| Minimum requirements                                                     |
++===================+======================================================+
+| Operating systems | Windows x64, Linux, Mac OS X (some work required)    |
++-------------------+------------------------------------------------------+
+| CPU               | Recent Intel or AMD cpus                             |
++-------------------+------------------------------------------------------+
+| RAM Memory        | 8 GB                                                 |
++-------------------+------------------------------------------------------+
+| Hard Drive        | ~400MB for Meshroom + space for your data            |
++-------------------+------------------------------------------------------+
+| GPU               | NVIDIA CUDA-enabled GPU (compute capability >= 2.0)  |
++-------------------+------------------------------------------------------+
 
+To obtain better performances on a desktop/laptop machine the recommended requirements are:
 
-**Minimum**
-: Ram: 8gb+, HDD: ~400MB Meshroom, 2gb+ for cache data and models
++--------------------------------------------------------------------------+
+| Recommended requirements                                                 |
++===================+======================================================+
+| CPU               | Intel Core i7 or AMD Ryzen 7                         |
++-------------------+------------------------------------------------------+
+| RAM Memory        | 32 GB                                                |
++-------------------+------------------------------------------------------+
+| Hard Drive        | 20GB+ HDD or SDD                                     |
++-------------------+------------------------------------------------------+
+| GPU               | NVIDIA GeForce GTX 1070                              |
++-------------------+------------------------------------------------------+
 
-CPU: not too old (~3 years and newer should be ok), NVIDIA GPU
 
+Software Requirements
+~~~~~~~~~~~~~~~~~~~~~
 
-**Recommended**
-: CPU: i7/Ryzen 7 or better, 32gb+ Ram, HDD/SSD: 20GB+ , NVIDIA GTX1070+
-(reconstruction time for this setup: 1 minute per image)
+If you are using the pre-built binaries the only requirements is python.
```

Comment:
Python is also embedded in the pre-built binaries.

Reply:
ok 994af28